spec/rubocop/cop/rails/read_write_attribute_spec.rb

```diff
@@ -16,4 +16,87 @@
     expect(cop.offenses.size).to eq(1)
     expect(cop.highlights).to eq(['write_attribute'])
   end
+
+  describe '#autocorrect' do
+    context 'write_attribute' do
```

Comment:
Instances with explicit receivers are not marked as offenses, and therefore also not autocorrected.

I could change it to also record them as offenses, and implement the according autocorrection. Otherwise I'd add some spec documenting the current behaviour.

What do you think?